frontend/components/MerchCard.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <div class="flex group flex-col items-center bg-white rounded shadow relative transition-all overflow-clip">
+        <div class="max-h-[350px] max-w-[350px] overflow-clip">
+            <svg width="29" height="29" viewBox="0 0 29 29" fill="none"
+                class="absolute z-30 left-2 top-2 opacity-0 transition-all group-hover:opacity-100"
+                xmlns="http://www.w3.org/2000/svg">
+                <circle cx="14.8203" cy="14.5" r="14" fill="white" />
+                <path
+                    d="M19.6203 19.3C20.0447 19.3 20.4516 19.4686 20.7517 19.7686C21.0517 20.0687 21.2203 20.4757 21.2203 20.9C21.2203 21.3243 21.0517 21.7313 20.7517 22.0314C20.4516 22.3314 20.0447 22.5 19.6203 22.5C19.196 22.5 18.789 22.3314 18.4889 22.0314C18.1889 21.7313 18.0203 21.3243 18.0203 20.9C18.0203 20.012 18.7323 19.3 19.6203 19.3ZM6.82031 6.5H9.43631L10.1883 8.1H22.0203C22.2325 8.1 22.436 8.18429 22.586 8.33431C22.736 8.48434 22.8203 8.68783 22.8203 8.9C22.8203 9.036 22.7803 9.172 22.7243 9.3L19.8603 14.476C19.5883 14.964 19.0603 15.3 18.4603 15.3H12.5003L11.7803 16.604L11.7563 16.7C11.7563 16.753 11.7774 16.8039 11.8149 16.8414C11.8524 16.8789 11.9033 16.9 11.9563 16.9H21.2203V18.5H11.6203C11.196 18.5 10.789 18.3314 10.4889 18.0314C10.1889 17.7313 10.0203 17.3243 10.0203 16.9C10.0203 16.62 10.0923 16.356 10.2123 16.132L11.3003 14.172L8.42031 8.1H6.82031V6.5ZM11.6203 19.3C12.0447 19.3 12.4516 19.4686 12.7517 19.7686C13.0517 20.0687 13.2203 20.4757 13.2203 20.9C13.2203 21.3243 13.0517 21.7313 12.7517 22.0314C12.4516 22.3314 12.0447 22.5 11.6203 22.5C11.196 22.5 10.789 22.3314 10.4889 22.0314C10.1889 21.7313 10.0203 21.3243 10.0203 20.9C10.0203 20.012 10.7323 19.3 11.6203 19.3ZM18.8203 13.7L21.0443 9.7H10.9323L12.8203 13.7H18.8203Z"
+                    fill="#636364" fill-opacity="0.5" />
+            </svg>
+            <div class="absolute right-2 top-2 bg-white rounded-md py-1 px-2 text-[#9E3500] z-30">{{ type }}</div>
+            <img :src="imageUrl"
+                alt="Merch Image" class="object-cover transition-all group-hover:scale-125 rounded-t">
+        </div>
+        <div
+            class="hidden opacity-0 transition-all group-hover:opacity-100 w-full text-center text-white z-20 poppins group-hover:block -translate-y-24">
+            View Product</div>
+        <div
+            class="absolute bottom-0 h-[50%] w-full bg-gradient-to-b from-transparent via-transparent to-black group-hover:-translate-y-36 transition-all">
+        </div>
+        <div
+            class="relative flex flex-col z-20 justify-start w-full gap-2 bg-white pt-2 px-3 pb-5 transition-all group-hover:-translate-y-20">
+            <div class="flex justify-between w-full text-base"><span class="text-[#9E3500] poppins font-bold">{{ seller
+                    }}</span> <span class="poppins font-bold">₹{{ price }}</span></div>
```

Comment:
The dev mode is unavailable for some reason. Also the font I checked is Poppins

![image](https://github.com/bsoc-bitbyte/GetIt/assets/76207818/f13c7c5f-8c47-40a4-a23c-a53b28ebafd8)


and Poppins is never imported throught out the project? To use a certain font even if it's not available on the computer you'll have to import it from Google fonts but i don't see Poppins or any other fonts imported. To fix this for now we can just import in the style tag but sometimes in a future pr implementing https://google-fonts.nuxtjs.org/getting-started/setup will be better.

Reply:
This isn't required for just this pr itself and will probably need change to few other files and the config, The font file needed for this pr itself ie poppins I have done that already through the import statement in the style tag.

Reply:
@Om-Thorat Why don't you add Poppins as the global Font for the project in this PR itself and make the necessary changes. 
Another thing to keep in mind is that avoid importing CDN since it increases page load time.

Reply:
Yea exactly why I suggested using the google-fonts nuxt, Sure then I'll make the necessary changes to the config and so to encorporate  https://google-fonts.nuxtjs.org/getting-started/setup .